src/vs/workbench/contrib/chat/common/chatServiceImpl.ts

```diff
@@ -246,6 +246,18 @@ export class ChatService extends Disposable implements IChatService {
 			}
 
 			const sessions = arrayOfSessions.reduce((acc, session) => {
+				// Revive serialized markdown strings in response data
+				for (const request of session.requests) {
+					if (Array.isArray(request.response)) {
```

Comment:
Does this need to handle sessions that were saved by an old version of vscode in the previous format where this was just a string?

Reply:
Uh wait nvm I misread what the previous type was

Reply:
Yes, thanks, fixed!